### DISCUSSION TITLE
Question about `gopass sync` and multiple remotes

### DISCUSSION THREAD

**Discussion by kartikdc** [gopasspw/gopass#2391] [Q&A]
Hi. This is a pretty rudimentary question. If the answer is in the docs please point me to them because I must have missed them.

When I do `gopass sync`, I get told that gopass will be syncing all remotes but only origin gets synced. I have to sync the rest (codeberg and sourcehut) manually.

Any idea what I am doing wrong? Or if I have to configure something so that all remotes are synced? I know that it is possible to set multiple push urls in a single remote that could work here but I was wondering if there is a simpler solution.

Thanks.

These are the remotes I have set up:

```
in ~ via Python v3.10.8 took 5s
 ❯ gopass git remote -v
⚠ Running 'git remote -v' in /home/kartik/.password-store...
codeberg        git@codeberg.org:***/***.git (fetch)
codeberg        git@codeberg.org:***/***.git (push)
origin  git@gitlab.com:***/***.git (fetch)
origin  git@gitlab.com:***/***.git (push)
sourcehut       git@git.sr.ht:~***/*** (fetch)
sourcehut       git@git.sr.ht:~***/*** (push)
```

**Comment by innovate-invent** [comment]
I believe the multiple push urls are the answer:
```
gopass git remote set-url --add origin git@codeberg.org:repos/pass.git
```
